src/sentry/tasks/base.py

```diff
@@ -129,6 +133,15 @@ def _wrapped(*args, **kwargs):
 
             return result
 
+        # If the split task router is configured for the task, always use queues defined
+        # in the split task configuration
+        if name in settings.CELERY_SPLIT_QUEUE_TASK_ROUTES:
+            q = kwargs.pop("queue")
+            if q:
+                logger.warning(
+                    "ignoring queue: %s, using value from CELERY_SPLIT_QUEUE_TASK_ROUTES", q
+                )
```

Comment:
I am not sure why we need this. 
Also this is going to be extremely verbose if you set the configuration first and only then, you remove the queue from the task.

Reply:
i added it because the queue name shouldn't be there and i think it promotes cleaning it up later. warnings aren't logged to sentry and it's only once on startup so i don't think it's very verbose